tests/manager/api/test_circulationapi.py

```diff
@@ -893,92 +851,23 @@ def test_fulfill_errors(self, circulation_api: CirculationAPIFixture):
         circulation_api.pool.open_access = True
         circulation_api.pool.collection = collection
 
-        circulation_api.circulation.remotes[
-            circulation_api.pool.data_source.name
-        ] = OPDSAPI(circulation_api.db.session, collection)
```

Comment:
This test didn't really make sense, because it was injecting a `OPDSAPI` into the `MockCirculationApi`. All the functionality that this tested in `OPDSAPI` is tested directly in that class, so most of this test was removed. The only part that was left is making sure when we call `FormatNotAvailable` bubbles up when calling `fulfill`.

The coverage for `OPDSAPI` shouldn't change at all with this removal.